### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Arnab-Developer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
 # test
-pr test
+Code review in PR
```
Comment (arnab-jit1) [inline README.md:2]: Change `PR` to `pull request`.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,2 @@
 # test
```
Comment (arnab-jit1) [inline README.md:1]: Make `t` cap.

**Review by arnab-jit1 — CHANGES_REQUESTED** [review]
Please fix

File: README.md (hunk 2 above)
Comment (Arnab-Developer) [reply README.md]: Done

File: README.md (hunk 1 above)
Comment (Arnab-Developer) [reply README.md]: Done

File: README.md [hunk 3]
```diff
@@ -1,2 +1,2 @@
-# test
-pr test
+# Test
+Code review in pull request
```
Comment (arnab-jit1) [inline README.md:2]: End with stop `.`

**Review by arnab-jit1 — CHANGES_REQUESTED** [review]
Please fix

**Review by arnab-jit1 — APPROVED** [review]
Good

**Pull request merged**